eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -98,9 +98,9 @@ jobs:
     # Linux x64
     - ${{ if eq(parameters.platform, 'linux_x64') }}:
       - ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        - Ubuntu.2204.Amd64.Open
+        - azurelinux.3.amd64.open
```

Comment:
should we add an Ubuntu container run so we don't loose coverage for that platform?

Reply:
The main benefit of the raw VM pattern is that it removes one layer of complexity (containers), e.g. there are small differences between running in the container or not like cgroup constraints etc. I'm also not sure whether it might have infrastructure benefits i.e. workitems start faster when they don't need to fetch the container image before running.

Question is whether that's enough reason to keep the existing setup.